proposals/description/fip_82b.ts

```diff
@@ -0,0 +1,463 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_82b: ProposalDescription = {
+  title: 'FIP-82b: Authorise the TribalCouncil with necessary roles',
+  commands: [
+    /////// Transfer all non-major role admins to the ROLE_ADMIN, to allow TribalCouncil to manage ////////
+    {
+      target: 'core',
+      values: '0',
+      method: 'createRole(bytes32,bytes32)',
+      arguments: [
+        '0xf0b50f04623eeaacfa1f202e062a3001c925a35c6b75d6903e67b43f44bbf152', // PARAMETER_ADMIN
+        '0x2172861495e7b85edac73e3cd5fbb42dd675baadf627720e687bcfdaca025096' // ROLE_ADMIN
+      ],
+      description: 'Transfer PARAMETER_ADMIN role admin to ROLE_ADMIN'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'createRole(bytes32,bytes32)',
+      arguments: [
+        '0xdc81827f5af6c7965785d62c38ca3481ccf540d7f561cac825518e49d6161c95', // RATE_LIMITED_MINTER_ADMIN
```

Comment:
RATE_LIMITED_MINTER_ADMIN doesn't appear anywhere in protocol or the permissions test.

However we would plan to use it with GlobalFeiMinter, so I'd just want to see it in permissions.ts with an empty list

Reply:
I test that it is correctly created and in the `dao/fip_82b.ts` validate script, will add to the permissions test.